pkg/ebpf/co_re.go

```diff
@@ -61,6 +61,7 @@ func (c *coreAssetLoader) loadCOREAsset(filename string, startFn func(bytecode.A
 		VerifierOptions: bpflib.CollectionOptions{
 			Programs: bpflib.ProgramOptions{
 				KernelTypes: btfData,
+				LogSize:     10 * 1024 * 1024,
```

Comment:
Did you see truncation in practice? The default is 64KiB, so increasing to 10MiB is quite a jump.

Reply:
Indeed: https://datadoghq.atlassian.net/browse/SUSM-51
Furthermore, this is the value specified in all other instances defining the log size